models/_generate_layers.py

```diff
@@ -344,6 +344,39 @@ def _parse_init_docstring(self):
   def __repr__(self):
     return f"<{self.__class__.__name__} {self.layer_class.__name__} {self.inspect_init_sig}>"
 
+  def get_super_call_code_str(self) -> str:
+    """
+    Inspects the super call of the layer class in RETURNN, extracts the super call parameters from there,
+    removes unwanted parameters and builds a super call which can be written as super call into _generated_layers.py
+    for that class.
+
+    :return: filtered parameters as kwargs_repr in a super call e.g `f"super().__init__({kwargs_repr})"`
```

Comment:
This is wrong.
It returns the code string for the super call.
```suggestion
    :return: sth like `super().__init__(...)`
```